### PR TITLE
[POC] Run tests with TiDb

### PR DESCRIPTION
**Pull request by sgiehl**
### Description:

We are experimenting with using TiDb as alterative to MySQL.

This PR is a temporary solution to run tests with TiDb instead of MySQL. It is not meant to merged, but only a placeholder to more easily find the latest test results.

### Review

* [ ] [Functional review done](https://developer.matomo.org/guides/pull-request-reviews#functional-review-done)
* [ ] [Potential edge cases thought about](https://developer.matomo.org/guides/pull-request-reviews#potential-edge-cases-thought-about) (behavior of the code with strange input, with strange internal state or possible interactions with other Matomo subsystems)
* [ ] [Usability review done](https://developer.matomo.org/guides/pull-request-reviews#usability-review-done) (is anything maybe unclear or think about anything that would cause people to reach out to support)
* [ ] [Security review done](https://developer.matomo.org/guides/security-in-piwik#checklist)
* [ ] [Wording review done](https://developer.matomo.org/guides/pull-request-reviews#translations-wording-review-done)
* [ ] [Code review done](https://developer.matomo.org/guides/pull-request-reviews#code-review-done)
* [ ] [Tests were added if useful/possible](https://developer.matomo.org/guides/pull-request-reviews#tests-were-added-if-usefulpossible)
* [ ] [Reviewed for breaking changes](https://developer.matomo.org/guides/pull-request-reviews#reviewed-for-breaking-changes)
* [ ] [Developer changelog updated if needed](https://developer.matomo.org/guides/pull-request-reviews#developer-changelog-updated-if-needed)
* [ ] [Documentation added if needed](https://developer.matomo.org/guides/pull-request-reviews#documentation-added-if-needed)
* [ ] Existing documentation updated if needed

### CODE REVIEW

File: core/Db/Schema.php [hunk 1]
```diff
@@ -221,4 +221,14 @@ public function addMaxExecutionTimeHintToQuery(string $sql, float $limit): strin
     {
         return $this->getSchema()->addMaxExecutionTimeHintToQuery($sql, $limit);
     }
+
+    /**
+     * Returns if the schema support complexe column updates
```
Comment (michalkleiner) [inline core/Db/Schema.php]:
```suggestion
     * Returns if the schema support complex column updates
```

Comment (sgiehl) [reply core/Db/Schema.php]: This PR is currently based on other PRs and all changes in here will be thrown away at some point, so reviewing doesn't make sense at all.
